src/dockerfiles/gpu/py310_env.yml

```diff
@@ -0,0 +1,17 @@
+name: py310_env
+channels:
+  - conda-forge
+dependencies:
+  - python==3.10.4
+  - numpy==1.22.2
+  - pandas==1.4.1
+  - pip==22.0.4
+  - scipy==1.7.3
+  - cloudpickle==2.0.0
+  - pyarrow==7.0.0
+  - boto3==1.18.0
+  - pydantic==1.9.0
+  - pip:
+    - scikit_learn==1.0.2
+    - typing_extensions==4.3.0
+    - aqueduct-ml==0.1.4
```

Comment:
These requirements are only needed for the docker build process right? The user's function requirements will overwrite these at execution time?